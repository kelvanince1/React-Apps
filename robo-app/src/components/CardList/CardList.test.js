import React from 'react';
import { shallow } from 'enzyme';

import CardList from './index';

it('Expect to render Card component', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'John Doe',
            username: 'Johnny',
            email: 'goJohnnygo'
        }
    ];
    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
