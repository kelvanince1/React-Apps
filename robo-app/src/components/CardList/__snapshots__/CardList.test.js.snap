// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Card List component 1`] = `ShallowWrapper {}`;

exports[`Expect to render Card component 1`] = `ShallowWrapper {}`;

exports[`Expect to render CardList component 1`] = `ShallowWrapper {}`;

exports[`Expect to render Cardist component 1`] = `ShallowWrapper {}`;
